app/src/main/resources/application.yml

```diff
@@ -3,12 +3,25 @@ server:
   ssl:
     enabled: false
 debug: false
+
 management:
+  security:
+    enabled: false
+  server:
+    port: ${MGMT_PORT:8081}
```

Comment:
Could you elaborate on the use case for this?

Reply:
By using a mgmt port it allows shutdown to be on by default, since presumably that port wouldn't be exposed externally (i.e. the internet). The mgmt port is only listening on the management address which defaults to 127.0.0.1 so it is only accessible locally. I don't know if the heroku deployment utilizes the health actuator and whether or not they support having the health check URL use a different port than the primary port of the app server, but I figured that is probably the only public deployment and that most uses of the initializr would be in a pipeline or when working with it locally. I can change the default b/c I normally run the shutdown as part of other scripts that could easily turn it on. Is moving the actuators to a management port going to be a problem or should I revert that too?

Reply:
Thanks for the explanation. No need to revert. So long as the interface is local, I don't think it would cause any issues.